engine/src/test/java/com/redhat/cloud/notifications/VulnerabilityTestHelpers.java

```diff
@@ -39,8 +38,8 @@ public static EmailAggregation createEmailAggregation(String bundle, String appl
                                                             .build())
                                     .build()
         ));
-        JsonObject payload = baseTransformer.toJsonObject(emailActionMessage);
-        aggregation.setPayload(payload);
+
+        aggregation.setPayload(TestHelpers.wrapActionToJsonObject(emailActionMessage));
```

Comment:
`baseTransformer` is no longer needed in this class.

Reply:
Removed!